99-antispam_with_sieve.conf

```diff
@@ -17,46 +17,33 @@ plugin {
   imapsieve_mailbox2_causes = COPY
   imapsieve_mailbox2_before = file:/usr/lib/dovecot/sieve/learn-ham.sieve
 
-  sieve_pipe_bin_dir = /usr/lib/dovecot/sieve
+  sieve_pipe_bin_dir = /usr/lib/dovecot/sieve-pipe
 
   sieve_global_extensions = +vnd.dovecot.pipe +vnd.dovecot.environment
+
   ##
-  #
   # rspamd:
   #
-  # X-Spamd-Bar: /
-  # X-Spamd-Result: default: False [0.00 / 150.00];
+  # X-Spamd-Result: default: False [0.00 / 150.00]; ...
   # X-Spam: Yes
 
   ##
-  #
```

Comment:
Why did you delete those? can we restore the examples for SA?

Reply:
the project is named "dovecot...rspamd". unrelated examples are imho not useful. what do you need them for?